src/config.rs

```diff
@@ -1,9 +1,11 @@
 use crate::net::constants::{FRAGMENT_SIZE_DEFAULT, MAX_FRAGMENTS_DEFAULT};
-use std::default::Default;
+use std::{default::Default, time::Duration};
 
 #[derive(Clone)]
 /// Struct that contains config values for various aspects of the network
-pub struct NetworkConfig {
+pub struct Config {
+    /// The maximal amount of time to keep `VirtualConnection`s around before cleaning them up.
```

Comment:
Probably wanted an apostrophe here?

Reply:
Wrapping in back ticks produces different markdown output so it's signifying a specific type. `VirtualConnection`